lib/BoundingBox.py

```diff
@@ -99,10 +99,10 @@ def getRelativeBoundingBox(self, imgSize=None):
                 'Parameter \'imgSize\' is required. It is necessary to inform the image size.')
         if imgSize is None:
```

Comment:
Hi, i feel little strange that why here imgSize is None, then use imgSize[0] and imgSize[1].
should be 'if imgSize is not None'??

Reply:
Yes you are right it should be `if imgSize is not None`. I corrected this mistake in my private fork but never committed it to this pull request because OP never didn't respond.

Reply:
I made the change in case this pull request is accepted.

Reply:
Thank you very much.

Reply:
@laclouis5,

Yes, you are completely right. Although it has no impact in the code, it should be fixed. 

Thank you for your contribution and sorry for the huge delay on checking the PRs.